db/migrate/20240722154234_create_customers_items.rb

```diff
@@ -0,0 +1,9 @@
+class CreateCustomersItems < ActiveRecord::Migration[7.1]
```

Comment:
Careful here, remember on the joins table, we want the first reference to be singular and the second reference to be pluralized so CustomerItems would be better.

Reply:
Ahhh, actually, I see what you did, you used [this syntax](https://guides.rubyonrails.org/association_basics.html#the-has-and-belongs-to-many-association:~:text=2.6%20The%20has_and_belongs_to_many%20Association), which is not wrong, it works just fine, it's just interesting because we don't teach it this way. The reason we don't teach it this way is because it abstracts what's going on under the hood with the foreign keys in the linking table. I'm fine if you use this syntax as long as you understand what's happening.